src/System.Private.CoreLib/src/System/ValueType.cs

```diff
@@ -26,6 +31,7 @@ public override String ToString()
             return this.GetType().ToString();
         }
 
+#if PROJECTN
```

Comment:
Should/could ProjectN adopt this?

Reply:
We would just have to update the IL2IL transform that does this. The long term plan is to get rid of transforms, so I would kind of prefer this was another of those things that will be naturally picked up when we do the next step of Project X (get rid of STS type system and host C2 from the CoreRT compiler driver directly).